frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/breakdown-card/breakdown-capacity.tsx

```diff
@@ -0,0 +1,11 @@
+import * as React from 'react';
+import './breakdown-card.scss';
+
+export const TotalCapacityBody: React.FC<TotalCapacityBodyProps> = ({ value, classname = '' }) => {
+  return <p className={`capacity-breakdown-card__capacity-body ${classname}`}>{value}</p>;
```

Comment:
use `classNames`
```suggestion
  return <p className={classNames('capacity-breakdown-card__capacity-body', className')}>{value}</p>;
```

and you also dont have to define default value for classname.

Reply:
ack